### DISCUSSION TITLE
Footnotes are not aligned with their labels

### DISCUSSION THREAD

**Discussion by jdknight** [pradyunsg/furo#428] [Potential Issue] [answered]
Footnote entries are not aligned:

![image](https://user-images.githubusercontent.com/1834509/164950428-fcf944c6-c530-4d5a-853a-195c469d6a13.png)

The first paragraphs inside a footnote are being applied a top-margin offset (`0.125rem`). If this offset is 
temporarily disabled:

```patch
diff --git a/src/furo/assets/styles/content/_lists.sass b/src/furo/assets/styles/content/_lists.sass
index 1bfbd138..76166c92 100644
--- a/src/furo/assets/styles/content/_lists.sass
+++ b/src/furo/assets/styles/content/_lists.sass
@@ -59,7 +59,9 @@ dl.glossary
       margin-right: 0.2rem
 
   dd
-    > p:first-child,
+    > p:first-child
+      margin-top: 0rem
+
     ul
       margin-top: 0.125rem
```

Footnote entries align nicely:

![image](https://user-images.githubusercontent.com/1834509/164950443-b46be099-4275-4073-be72-8c246879a886.png)

Can we tweak the styling logic a bit so that footnote entries do not have this offset? (assuming there is a reason for other entries to have said offset)

**Comment by jab** [comment]
In case it makes sense to look at them both at the same time, I just reported a new issue with footnote misalignment in https://github.com/pradyunsg/furo/discussions/452.

**Comment by pradyunsg** [comment, marked as answer]
A PR for this would be welcome.
